src/components/Worktray/Worktray.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+
+/**
+ * The WorkTray component, that can be used to display a user's current work items.
+ *
+ * @noInheritDoc
+ */
+
+export interface WorkTrayProps {
+  /**
+   * Array of {@link WorkTrayRow} that defines the number of rows within the Work Tray
+   */
+  rows: WorkTrayRow[];
+  /**
+   * Array of {@link WorkTrayColumn} that defines the number of columns within the Work Tray
+   */
+  columns: WorkTrayColumn[];
+  /**
+   * Callback function used to reassign a work item to another user. If this function returns true, the item will be removed from the work tray.
+   */
+  reassignWorkItem: (workItemId: string) => boolean;
+  /**
+   * Callback function used to cancel a work item. If this function returns true, the item will be removed from the work tray.
+   */
+  cancelWorkItem: (workItemId: string) => boolean;
+  /**
+   * Callback function used to pass a search query back to your API and update the data being passed to {@link WorkTrayProps.rows}
+   */
+  searchWorkItems: (searchTerm: string) => void;
+}
+
+export interface WorkTrayRow {
+  /**
+   * Array of {@link WorkTrayItem} that defines the data that will be displayed in a particular row on the Work Tray
+   */
+  row: WorkTrayCell[];
+  /**lxs
+   * When a {@link WorkTrayColumn.key} is provided, an icon marking a Cautionary Contact will be display in that cell for the row.
+   */
+  cautionaryContactKey?: string;
```

Comment:
There was some discussion about whether this should be in the generic component as it's specific to housing. The suggested alternative was to instead show the alert only in the work item's detail view. We should get some feedback from Gill on this as I can see it would potentially be useful to have the alert flag in the summary view too.